src/App.jsx

```diff
@@ -1,72 +1,124 @@
 import useLocalStorageState from "use-local-storage-state";
+import { useState, useEffect } from "react";
 import { uid } from "uid";
-import { initialColors } from "./lib/colors";
+import { initialThemes } from "./lib/themes";
 import "./App.css";
-import Color from "./Components/Color/Color";
-import ColorForm from "./Components/ColorForm/ColorForm";
+import ThemeSelection from "./Components/ThemeSelection/ThemeSelection";
+import Theme from "./Components/Theme/Theme";
 
 function App() {
-  const [colors, setColors] = useLocalStorageState("colors", {
-    defaultValue: initialColors,
+  const [themes, setThemes] = useLocalStorageState("themes", {
+    defaultValue: initialThemes,
   });
+  const [currentTheme, setCurrentTheme] = useState(themes[0]);
```

Comment:
just a suggestion to use state with initial value : const [currentTheme, setCurrentTheme] = useState(initialThemes[0]);..I am not sure if it works  then but talking in general

Reply:
I have now combined it:

```
const [currentTheme, setCurrentTheme] = useState(
    themes[0] || initialThemes[0]
  );
  ```